code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -587,11 +587,27 @@
 	icon_state = "portafreezer"
 	item_state = "medicalpack"
 	max_w_class = 3
-	can_hold = list(/obj/item/organ, /obj/item/reagent_containers/food, /obj/item/reagent_containers/glass)
 	max_storage_space = 21
-	use_to_pickup = 1 // for picking up broken bulbs, not that most people will try
+	use_to_pickup = 0 // for picking up broken bulbs, not that most people will try
 	chewable = FALSE
 
+	/obj/item/storage/box/freezer/organcooler
+	name = "organ cooler"
+	desc = "A sealed, cooled container to keep organs from decaying."
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "organcooler"
+	item_state = "firstaid-advanced"
+	max_w_class = 3
```

Comment:
```suggestion
	max_w_class = ITEMSIZE_NORMAL
```

Reply:
Was not aware this was a thing. Is this the better way of defining item sizes? I noticed tinkering with sizes above 5 all designated them as huge, or was it bulky. The big one!

Reply:
It is better to use defines where they exist, yeah. Main reason is to make the code more readable & make it easier if anyone decides to edit the value of `ITEMSIZE_NORMAL` one day, reduces the amount of modifications they'll need.

Reply:
ic ic! Sweet!